tests/integration/generate_fixtures/common.py

```diff
@@ -34,22 +34,28 @@
     "config": {
         "chainId": 131277322940537,  # the string 'web3py' as an integer
         "homesteadBlock": 0,
+        "byzantiumBlock": 0,
+        "constantinopleBlock": 0,
+        #  "petersburgBlock": 0,
```

Comment:
I left these in because docs for syncing a parity + geth private net is woefully sparse. So, this is part documentation + part todo list. Does that context change anything or would you prefer that it live in an issue? No biggie either way.

Reply:
Fair enough. I think it's fine here!